### ISSUE TITLE
Unable to access vuex-orm $db variable from store in nuxt

### ISSUE THREAD

**Issue by flyingboy007** [vuex-orm/vuex-orm#548]
Using nuxtjs with vuexorm & vuexorm-axios plugin. 

> /pages/index.vue

    computed: {
      users() {
     // this works as expected
      return User.all()
      }
        }

> plugins/vuex-orm-axios.js

    import { Model } from '@vuex-orm/core'
    
    export default ({ $axios }) => {
      Model.setAxios($axios)
    }

> store/index.js

    import VuexORM from '@vuex-orm/core'
    import VuexORMAxios from '@vuex-orm/plugin-axios'
    import User from '@/models/user'
    
    
    VuexORM.use(VuexORMAxios)
    
    
    // Create a new database instance.
    const database = new VuexORM.Database()
    
    // Register Models to the database.
    database.register(User)
    
    
    export const plugins = [
      VuexORM.install(database)
    ]

Above all works. But in vuexorm [docs][1] it says to always fetch model from injected database instance for nuxt/ssr apps.
But if I try to access the `$db` variable from `store` it wont work as there is no $db variable inside store. 

> /pages/index.vue

    computed: {
      // this wont work as $db is undefined
        User () {
         return this.$store.$db().model('users')
        },
        users () {
         return this.User.all()
        }
     
        }

In short calling `this.$store `works. But there is no $db instance in store variable. `this.$store.$db() `returning `undefined` 
What am I doing wrong here?

  [1]: https://vuex-orm.github.io/vuex-orm/guide/digging-deeper/server-side-rendering.html

**Comment by cuebit** [comment]
@flyingboy007 you seem to have User and users within a users method. Is this a typo?

**Comment by flyingboy007** [comment]
> @flyingboy007 you seem to have User and users within a users method. Is this a typo?

Was a typo. Sorry for that. Fixed

**Comment by cuebit** [comment]
By way of update @flyingboy007, there is a known issue when using Vuex ORM `0.35.x` with Nuxt so it may be this is a conflicting issue with #544

Will keep you posted.

**Comment by kiaking** [comment]
@flyingboy007 #544 is fixed with Vuex ORM 0.36.1. Could you try this one and see if it fixed this one as well?

**Comment by flyingboy007** [comment]
@kiaking Great I will try and update

**Comment by flyingboy007** [comment]
Thank you for the fix. It is working now.

**Comment by kiaking** [comment]
@flyingboy007 Wonderful to hear that!!! Thank you for confirming 👍
